src/DssSpell.t.base.sol

```diff
@@ -598,6 +598,11 @@ contract DssSpellTestBase is Config, DssTest {
                         assertTrue(abi.decode(value, (uint256)) > 0 && abi.decode(value, (uint256)) < RAY, _concat("TestError/calc-cut-range-", ilk));
                     }
                 }
+                {
+                    uint256 _chost = (values.collaterals[ilk].dust * RAD) * ((values.collaterals[ilk].chop * WAD / 10000) + WAD) / WAD;
```

Comment:
```suggestion
                    uint256 normalizedTestChop = (values.collaterals[ilk].chop * 10**14) + WAD;
                    uint256 _chost = (values.collaterals[ilk].dust * RAD) * normalizedTestChop / WAD;
```

Reply:
neat - this is more similar to other places in this file where chop is normalized.